agent/qrexec-agent-data.c

```diff
@@ -220,9 +226,9 @@ int handle_remote_data(libvchan_t *data_vchan, int stdin_fd, int *status,
     while (libvchan_data_ready(data_vchan) > 0) {
         if (libvchan_recv(data_vchan, &hdr, sizeof(hdr)) < 0)
             return -1;
-        if (hdr.len > MAX_DATA_CHUNK) {
-            fprintf(stderr, "Too big data chunk received: %d > %d\n",
-                    hdr.len, MAX_DATA_CHUNK);
+        if (hdr.len > max_len) {
+            fprintf(stderr, "Too big data chunk received: %d > %lu\n",
```

Comment:
`hdr.len` is `uint32_t` and `max_len` is `size_t` so `%" PRIu32 "` and `%zu` are the correct formats, I think.